test/fixture/testrepos/nginx.conf

```diff
@@ -13,8 +13,38 @@ http {
     auth_basic_user_file  .htpasswd;
     root /tmp/argo-e2e;
 
-    ssl_certificate localhost.crt;
-    ssl_certificate_key localhost.key;
+    ssl_certificate ../certs/argocd-test-server.crt;
+    ssl_certificate_key ../certs/argocd-test-server.key;
+    ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
+   
+    location ~ /argo-e2e(/.*) {
+      # Set chunks to unlimited, as the body's can be huge
+  	  client_max_body_size			0;
+
+  	  fastcgi_param	SCRIPT_FILENAME		/usr/lib/git-core/git-http-backend;
+  	  include		/etc/nginx/fastcgi_params;
+  	  fastcgi_param	GIT_HTTP_EXPORT_ALL	"";
+  	  fastcgi_param	GIT_PROJECT_ROOT	/tmp/argo-e2e;
```

Comment:
could this be `/tmp/argocd-e2e`

Reply:
Hm, I think throughout E2E tests, `/tmp/argo-e2e` is used - so this would be some major refactoring. But you're right - just let's do this in another PR.